src/app/components/Card/Card.tsx

```diff
@@ -0,0 +1,99 @@
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import Paper, { PaperProps } from '@mui/material/Paper'
+import Divider from '@mui/material/Divider'
+import useTheme from '@mui/material/styles/useTheme'
+import Typography from '@mui/material/Typography'
+import { darken, lighten } from '@mui/material/styles'
+
+import { ICard, ICrop, IPlayedCrop } from '../../../game/types'
+import { isCrop } from '../../../game/types/guards'
+import { cards, isCardImageKey, ui } from '../../img'
+
+import { CardCropText } from './CardCropText'
+
+export type CardProps = PaperProps & {
+  card: ICard
+  size?: number
+} & {
+  card: ICrop
+  playedCrop?: IPlayedCrop
+}
+
+export const Card = ({
+  card,
+  playedCrop,
+  size = 0.75,
+  sx = [],
+  ...rest
+}: CardProps) => {
+  const theme = useTheme()
+
+  const imageSrc = isCardImageKey(card.id) ? cards[card.id] : ui.pixel
+
+  if (imageSrc === ui.pixel) {
+    console.error(`Card ID ${card.id} does not have an image configured`)
+  }
+
+  return (
+    <Paper
+      sx={[
+        {
+          width: 300,
+          height: 500,
```

Comment:
just thinking out loud here, but i wonder if we should size these using `rem` units, or something generally more flexible that static px values